src/Sylius/Behat/Context/Api/Admin/ManagingShippingMethodsContext.php

```diff
@@ -227,6 +236,25 @@ public function iSwitchTheWayShippingMethodsAreSortedByName(): void
         $this->client->sort('translation.name', 'desc');
     }
 
+    /**
+     * @When I filter archival shipping methods
+     */
+    public function iFilterArchivalShippingMethods(): void
+    {
+        $this->client->addFilter('exists[archivedAt]', true);
+        $this->client->filter();
+    }
+
+    /**
+     * @When I am browsing archival shipping methods
+     */
+    public function iAmBrowsingArchivalShippingMethods(): void
+    {
+        $this->client->index();
+        $this->client->addFilter('exists[archivedAt]', true);
```

Comment:
I'm wondering, maybe we should have another API client just for index? Or filtering? Doing so, we could make more specified methods and skip technical details in the context 💃

Reply:
good idea 🎉